Core/Core/Login/LoginUsePolicy/LoginUsePolicyView.swift

```diff
@@ -0,0 +1,96 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2023-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import SwiftUI
+
+struct LoginUsePolicyView: View {
+
+    @ObservedObject var viewModel: LoginUsePolicyViewModel
+    @Environment(\.viewController) var controller
+    @Environment(\.appEnvironment) var env
+
+    var body: some View {
+        VStack(spacing: 0) {
+            Text("Either you're a new user or the Acceptable Use Policy has changed since you last agreed to it. Please agree to the Acceptable Use Policy before you continue.", bundle: .core)
+                .font(.regular16).foregroundColor(.textDarkest)
+                .padding(EdgeInsets(top: 24, leading: 16, bottom: 16, trailing: 16))
+            Divider().padding(.zero)
+            Button {
+                env.router.route(to: "/accounts/self/terms_of_service", from: controller)
```

Comment:
We could move this to the viewmodel file and get rid of the env variable in the view.